engine/combat/utility/sos_appraisal.py

```diff
@@ -54,22 +54,32 @@ class SoSResource(Enum):
 
 
 class SoSAppraisal(Appraisal):
+    MAX_ENEMY_TARGETING_FAILURES = 6
```

Comment:
There are some fights with more targets I think? ie Catalyst, Gun Goddess and so on. Ant Bruisers if unlucky.

Reply:
I hope to fix this by the time we have any major issues with this - this is just a monkey patch for now